app/assets/javascripts/application.js

```diff
@@ -113,6 +113,15 @@ function ShowHideContent() {
 
 $(document).ready(function() {
 
+  // Warn about using the kit in production
+  if (
+    window.sessionStorage && window.sessionStorage.getItem('prototypeWarning') !== 'false' &&
+    window.console && window.console.info
+  ) {
+    window.console.info('GOV.UK Prototype Kit - do not use for production');
+    window.sessionStorage.setItem('prototypeWarning', true);
```

Comment:
Presumably you want this to be set to `'false'`, right? Or maybe `'shown'` or something more descriptive.